.github/workflows/build-wasm.yml

```diff
@@ -10,27 +10,15 @@ on:
       - 'LICENSE'
       - 'CODEOWNERS'
       - 'renovate.json'
+      - 'query-engine/**'
 
 jobs:
   build:
-    name: "WebAssembly build ${{ github.event.ref }} for commit ${{ github.event.inputs.commit }}"
+    name: "prisma-fmt-wasm build ${{ github.event.ref }} for commit ${{ github.event.inputs.commit }}"
     runs-on: buildjet-16vcpu-ubuntu-2004
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          target: wasm32-unknown-unknown
-          default: true
+      - uses: cachix/install-nix-action@v16
 
-      - name: Build the migration-connector crate with default features
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: -p migration-connector --release --target=wasm32-unknown-unknown
-
-      - name: Build the prisma-fmt crate
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: -p prisma-fmt --lib --release --target=wasm32-unknown-unknown
```

Comment:
This was just a confidence measure that these build do not fail - and now we are running similar builds via the `nix` things?

Reply:
`prisma-fmt` and all the psl compiling on wasm is implicitly validated by having a working build of `prisma-fmt-wasm`. `migration-connector` doesn't have exports that would translate to wasm exports so I think that check wasn't checking much in the first place. I'd prefer writing something proper that actually tests compiled-to-wasm code rather than trust what `cargo build` tells us.